ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/DebugAccountAt.java

```diff
@@ -81,6 +81,7 @@ protected Object resultByBlockHash(
           requestContext.getRequest().getId(), JsonRpcError.BLOCK_NOT_FOUND);
     }
 
+    // transactions.isEmpty
```

Comment:
Is this line commented on purpose or was it just for testing?

Reply:
removed